bin/makeMarkerStats.R

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env Rscript
```

Comment:
out of curiosity, as we can still distribute these on the cluster, how long does it take to do one of the large experiments through this?

Reply:
I'll let you know when I try it on a big one.

But note that I make the problem size as small as possible by selecting to the top n genes before calculating stats.